### PR TITLE
Expose more styles from GOV.UK modules

### PR DESCRIPTION
**Pull request by gavboulton**
Organise images by source:
* Move `passports` directory to `hmpo` and store all images from GOV.UK sources in `govuk`. Use the `$path` to set the image paths as sass modules are loaded.

Update validation to use `.error-summary`:
* Styling for `.validation-summary` has changed to `.error-summary`. Update the validation JS module that uses this class to add focus when form errors exist.

### CODE REVIEW

**Comment by easternbloc** [comment]
As this is used by wider HO is there an opportunity here to start dropping the 'passports' / 'hmpo' and just using ho in naming conventions? What do you think?

**Comment by gavboulton** [comment]
'hmpo' matches the package name.

**Comment by easternbloc** [comment]
we could rename the package?

**Comment by gavboulton** [comment]
We could but I don't think that's necessary at the moment.

**Comment by gavboulton** [comment]
@HughePaul @oli-weighell2 @JoeChapman @joefitter @daniel-ac-martin?

**Comment by daniel-ac-martin** [comment]
@gavboulton: What the reason for the .validation-summary to .error-summary change? Is that something that has changed upstream in elements?

File: assets/stylesheets/_typography.scss [hunk 1]
```diff
@@ -20,8 +20,8 @@ strong {
 
 .new-window {
     padding-right: 17px;
-    background: file-url("passports/new-window-link.png") right center no-repeat;
+    background: file-url("new-window-link.png") right center no-repeat;
     &:hover {
-        background-image: file-url("passports/new-window-link-blue.png");
+        background-image: file-url("new-window-link-blue.png");
```
Comment (daniel-ac-martin) [inline assets/stylesheets/_typography.scss]: So has this particular image moved up a directory rather than into the `hmpo` dir?

Comment (gavboulton) [inline assets/stylesheets/_typography.scss]: The `file-url` function uses the `$path` variable that's set before these sass modules are included.

**Comment by gavboulton** [comment]
Yes @daniel-ac-martin, `.validation-summary` has changed to `.error-summary` in GOVUK Elements.

File: assets/stylesheets/_typography.scss (hunk 1 above)
Comment (daniel-ac-martin) [inline assets/stylesheets/_typography.scss]: Ah, sorry Gav, I missed that one!

**Comment by daniel-ac-martin** [comment]
LGTM!

**Comment by oli-weighell2** [comment]
Readme update - https://github.com/UKHomeOffice/passports-frontend-toolkit/tree/feature/increase-styles#images. Otherwise, LGTM.

**Comment by oli-weighell2** [comment]
Also, tweak to the readme for passports-templates - https://github.com/UKHomeOffice/passports-templates#templates

**Comment by gavboulton** [comment]
@oli-weighell2 Is the update to passports-templates to do with the frontend toolkit or to update and add to the template list?

**Comment by oli-weighell2** [comment]
Was thinking of the template list.

**Comment by gavboulton** [comment]
README updated.

**Pull request merged**